docs/index.md

```diff
@@ -1,4 +1,6 @@
-![EESSI logo](img/logos/EESSI_logo_horizontal.png)
+<p align="center">
+  <img width="60%" src="img/logos/EESSI_logo_horizontal.png" alt="EESSI logo">
```

Comment:
Did you test this, does it work in a deployed version of the docs?

Reply:
See also https://github.com/mkdocs/mkdocs/issues/991 (not sure if that's fixed)

Reply:
Nope, didn't test, assumed the CI would that for me.

If there's a related bug, I can just use an absolute link

Reply:
It rendered fine in the editor

Reply:
According to [this comment](https://github.com/mkdocs/mkdocs/issues/991#issuecomment-233213340) it should work fine

Reply:
Tested, works